style/sass/.scss-lint.yml

```diff
@@ -181,6 +181,11 @@ linters:
     enabled: true
     style: one_space
 
+  SpaceAfterComment:
+    enabled: true
+    style: one_space
```

Comment:
Does this require a space after comments' `//`? I wonder if this would conflict with sassdocs's `///` syntax.